Sushi.Mediakiwi.Data.Elastic/Extensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Sushi.Mediakiwi.Data.Elastic
+{
+    public static class Extensions
+    {
+        /// <summary>
+        /// Adds a singleton instance for <see cref="Data.Repositories.INotificationRepository" /> to the service collection, which uses Elastic as backingstore.
+        /// </summary>
+        /// <param name="services"></param>
+        /// <param name="client"></param>
+        /// <param name="dataStreamName"></param>
+        public static void AddElasticNotifications(this IServiceCollection services, Nest.IElasticClient client, string dataStreamName = "notifications")
+        {
+            var repository = new Repositories.NotificationRepository(client, dataStreamName);
+            services.AddSingleton<Data.Repositories.INotificationRepository>(repository);
+            services.AddSingleton(repository);
```

Comment:
Why add two singletons ?

Reply:
Its the same instance. If anybody doesn't care and just wants a 'INotificationsRepository', they get this elastic instance. If anybody specificallly wants a ElasticNotificationsRepo, they also get this instance.

The second case is in use for the Elastic.NotificationsLists